### ISSUE TITLE
Path "[...]/node_modules/ng2-charts/schematics/src/collection.json" does not exist.

### ISSUE THREAD

**Issue by Lyro1** [valor-software/ng2-charts#1887]
**Reproduction of the problem**

- Create an Angular 17 project (running on Node.js v20)
- Try to include ng2-charts on this project using `ng add ng2-charts` or `ng add ng2-charts --save`.

**Result**:
```
 ~/[...]-Landing/ [main+*] ng add ng2-charts --save    
ℹ Using package manager: npm
✔ Found compatible package version: ng2-charts@6.0.0.
✔ Package information loaded.

The package ng2-charts@6.0.0 will be installed and executed.
Would you like to proceed? Yes
✔ Packages successfully installed.
Path "[...]-Landing/node_modules/ng2-charts/schematics/src/collection.json" does not exist.
```

Is there a simple way to fix this?

**Comment by wonaGuillermo** [comment]
Hello good afternoon, I have the same problem with NodeJS version 18.13.0 and Angular 17.3.2. Is there any solution?

**Comment by Alekevin12** [comment]
Hi @santam85, I had the same problem.

Iv'e digged a little deeper i noticed that the npm package it's missing the entire ng2-charts-schematics build, so I've cloned the repo and builded it locally, it seems that the build of the demo app is removing the schematics folder.

I've created a pr #1897 

Sorry if something's wrong but it's my first pr 😅

**Comment by santam85** [comment]
This is fixed in 6.0.1, thanks @Alekevin12
